dags/ods/kktix_ticket_orders/udfs/kktix_loader.py

```diff
@@ -0,0 +1,65 @@
+import json
+import os
+from typing import Dict, List
+
+import pandas as pd
+from google.cloud import bigquery
+from ods.kktix_ticket_orders.udfs.bigquery_loader import TABLE
+
+SCHEMA = [
+    bigquery.SchemaField("id", "INTEGER", mode="REQUIRED"),
+    bigquery.SchemaField("name", "STRING", mode="REQUIRED"),
+    bigquery.SchemaField("attendee_infos", "STRING", mode="REQUIRED"),
+]
+JOB_CONFIG = bigquery.LoadJobConfig(schema=SCHEMA)
+
+
+def load(event_raw_data_array: List):
+    """
+    load data into bigquery!
+    """
+    # data quality check
+    for event_raw_data in event_raw_data_array:
+        if len(event_raw_data["attendee_infos"]) == 0:
+            print("Nothing to load, skip!")
+            return
+    _load_to_bigquery(event_raw_data_array)
+    _load_to_klaviyo()
```

Comment:
It’s looks like that the data (e-mail) had been transformed (hashed) here, but we need e-mail address raw data.

Reply:
Yea you're right, commented that functuon below~